modules/cli/src/main/java/org/apache/ignite/internal/cli/commands/Options.java

```diff
@@ -336,9 +336,9 @@ public static final class Constants {
         /** Config update file option description. */
         public static final String CONFIG_UPDATE_FILE_OPTION_DESC = "Path to file with config update commands to execute";
 
-        public static final String RECALCULATE_ZONE_NAMES_OPTION = "--zone-names";
+        public static final String RESET_ZONE_NAMES_OPTION = "--zone-names";
 
-        public static final String RECALCULATE_ZONE_NAMES_OPTION_DESC = "Comma-separated list of zone names to recalculate data nodes for. "
-                + "If not specified, recalculates for all zones.";
+        public static final String RESET_ZONE_NAMES_OPTION_DESC = "Comma-separated list of zone names to reset data nodes for. "
```

Comment:
`RESET_ZONE_NAMES_OPTION` -> `RESET_DATA_NODES_ZONE_NAMES_OPTION`
`RESET_ZONE_NAMES_OPTION_DESC` -> `RESET_DATA_NODES_ZONE_NAMES_OPTION_DESC`

because it may confuses with zone partitions or something related to a zone reset